spec/function.dd

```diff
@@ -836,13 +837,13 @@ $(H2 $(LNAME2 optional-parenthesis, Optional Parentheses))
             fun;        // OK
 
             int[] arr;
-            arr.bar();  // OK
+            arr.bar();  // UFCS call
             arr.bar;    // also OK
         }
         ---
         )
 
-    $(P Optional parentheses are not applied to delegates or function pointers.)
+    $(P Note that parentheses are required to call a delegate or a function pointer:)
```

Comment:
Maybe squeeze a bit more info in there.

Reply:
`auto val = functionPtr;` might either copy the function pointer, or call it and store the result. Regular functions don't have this problem since you can't treat them as values.